src/tests/end-to-end/tests/popup/__snapshots__/launchpad.test.ts.snap

```diff
@@ -103,7 +103,7 @@ exports[`Popup -> Launch Pad content should match snapshot 1`] = `
               >
                 <div
                   aria-hidden="true"
-                  class="ms-Grid-col ms-sm3 popup-start-dialog-icon-circle"
+                  class="ms-Grid-col ms-sm000 popup-start-dialog-icon-circle"
```

Comment:
This is a side effect of the changes I made on the office fabric class name normalizer. It's unfortunate as this classes are related to the office fabric grid and are manually added.
Getting the regexp to skip this classes would add a lot of more complexity than I would like to add. Also, we should move away from office fabric grid and favor CSS grid and flex box instead.
Base on those arguments, I think it's fine to live with this small side effect for now.